dist/index.js

```diff
@@ -202,26 +217,26 @@ module.exports = function(token, orgId, apiHost) {
   }
 
   /**
-   * Notifies Humanitec that a build has completed
-   * @param {string} imageName - The name of the image to be added to Huamnitec.
-   * @param {Payload} payload - Details about the image.
+   * Notifies Humanitec that a version has been added
+   * @param {Payload} payload - Details about the artefact version.
    * @return {Promise} - A promise which resolves to true if successful, false otherwise.
    */
-  function addNewBuild(imageName, payload) {
+  function addNewVersion(payload) {
     return fetch(
-      `https://${apiHost}/orgs/${orgId}/modules/${imageName}/builds`, {
+      `https://${apiHost}/orgs/${orgId}/artefact-versions`, {
         method: 'POST',
         headers: {
           'Authorization': `Bearer ${token}`,
           'Content-Type': 'application/json',
+          'User-Agent': 'gh-action-build-push-to-humanitec/version',
```

Comment:
Sorry `/version` should be replaced by the version of action or maybe just `/latest` if getting  the action isn’t easy

Reply:
ahahah, sorry I am a total idiot! I thought you want to specify that it was the version of the action that adds versions instead of build!!!

Reply:
It wouldn't make any sense anyway (as we are hitting the version endpoint) but that was I thought when I wrote it 🤦‍♀️